LocalConfiguration.hpp

```diff
@@ -6,8 +6,8 @@
 // Don't load local configuration file
 #elif defined(MATRIX_LOCAL_CONFIG)
     #include "Configuration_local_matrix.hpp"
-#elif BOARD == BOARD_AVR_MEGA2560 && __has_include("Configuration_local_mega2560.hpp")
-    #include "Configuration_local_mega2560.hpp"
+#elif BOARD == BOARD_AVR_RAMPS && __has_include("Configuration_local_ramps.hpp")
```

Comment:
Why retire the MEGA2560?

Reply:
Since you can't actually build for it any more (@CTetford removed it in #163 ), it was fully replaced with the ramps build